contacts/emails_method_setting.py

```diff
@@ -0,0 +1,38 @@
+# gmail settings
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'youremail@gmail.com'
+EMAIL_HOST_PASSWORD = 'yourpassword'
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+
+
+#sendgrid settings
+EMAIL_HOST = 'smtp.krdia hoa' \
```

Comment:
similarly, use mailtrap.

Reply:
no this is unnecessary file by mistake push sorry